Source/WTF/wtf/StdLibExtras.h

```diff
@@ -863,7 +864,7 @@ ALWAYS_INLINE decltype(auto) makeUnique(Args&&... args)
 }
 
 template<class T, class... Args>
-ALWAYS_INLINE decltype(auto) makeUniqueWithoutRefCountedCheck(Args&&... args)
+ALWAYS_INLINE ConstUniquePtr<T> makeUniqueWithoutRefCountedCheck(Args&&... args)
```

Comment:
Probably worth leaving behind a comment that explains that this function should really only be used when a parent object points to a child object that will ref/deref the parent.

Reply:
Ok, suggesting:
```
// This function is useful when constructing an object that is forwarding its ref-counting to its
// owner. The function returns a `const std::unique_ptr<>` so that it cannot be reassigned. In
// case of reassignment, ref-counting forwarding wouldn't be safe. This function is commonly used
// with `lazyInitialize()` to initialize a const data member.
template<class T, class U = T, class... Args>
ALWAYS_INLINE const std::unique_ptr<U> makeUniqueWithoutRefCountedCheck(Args&&... args)
```

Reply:
We may also consider changing the name of `makeUniqueWithoutRefCountedCheck` but I'm keeping this out of this PR because it's hard to come up with a good name.